scormxblock/static/sass/scormxblock.scss

```diff
@@ -0,0 +1,193 @@
+        margin-left: 14px;
+        padding-top: 5px;
+
+        span {
+            font-weight: bold;
+        }
+    }
+
+    .full-screen-scorm {
+        position: fixed;
+        border: none;
+        overflow: hidden;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        z-index: 9999999999;
+        background-color: #fff;
+
+        .full-screen-off {
+            display: block;
+            margin: 10px;
+            position: absolute;
+            right: 20px;
+        }
+
+        .full-screen-on {
+            display: none;
+        }
+
+        .scorm_object {
+            position: fixed;
+            border: none;
+            overflow: hidden;
+            top: 55px;
+            right: 0;
+            bottom: 0;
+            left: 0;
+            width: 100%;
+            height: calc(100% - 65px);
+            z-index: 9999999;
+        }
+    }
+
+    .full-screen-off {
+        display: none;
+    }
+
+    .scorm-structure {
+        width: 300px;
+        margin-right: 0;
+
+        @media (max-width: 600px) {
+            padding: 0 25px 15px 15px;
+            width: 90%;
+        }
+    }
+
+    .scorm-structure-navigation {
+        max-width: 200px;
+        height: 450px;
+        overflow: auto;
+        margin: 0;
+        padding-left: 0 !important;
+
+        @media (max-width: 600px) {
+            height: auto;
+            max-width: 100%;
+        }
+
+        &-head {
+            color: #333;
+            font: 600 14px/20px Montserrat,"Helvetica Neue",Helvetica,Arial,sans-serif;
+            letter-spacing: .4px;
+            cursor: pointer;
+            list-style: none;
+            margin: 0 !important;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            position: relative;
+            white-space: nowrap;
+            padding-left: 4px;
+
+            @media (max-width: 600px) {
+                margin-bottom: 10px !important;
+            }
+
+            &.hide .icon-arrow {
+                transform: rotate(0deg);
+            }
+
+            .icon-folder {
+                position: relative;
+                top: 3px;
+            }
+
+            .icon-arrow {
+                margin: 0 2px 2px 0;
+                transform: rotate(90deg);
+            }
+        }
+
+        &-body {
+            margin: 5px 0 15px !important;
+
+            li {
+                list-style: none;
+                margin-bottom: 0 !important;
+
+                @media (max-width: 600px) {
+                    margin-bottom: 10px !important;
+                }
+
+                a {
+                    font-size: 12px;
+                    line-height: 14px;
+                }
+
+                a:hover {
+                    color: #0075b4 !important;
+                    text-decoration: underline !important;
+                }
+
+                a.current {
+                    color: #333;
```

Comment:
This color code matches 4 times, please add it like a variable.

Reply:
Done